### PR TITLE
Correcting Optional type hints in animal_requirements.py

### PR DESCRIPTION
**Pull request by Fjordrunner**
This is a short pull request dedicated to changing some optional type hints in animal_requirements.py to the correct format.

### CODE REVIEW

**Review by krb78 — APPROVED** [review]
Looks good from here! Nice and consistent. Thanks, Simon!

File: RUFAS/routines/animal/ration/animal_requirements.py [hunk 1]
```diff
@@ -1763,7 +1763,8 @@ def energy_activity_rqmts(self, body_weight: float, housing: str, distance: Opti
         Notes
         -----
         Activity requirement (net_energy_activity) is proportional to body weight and daily walking distance.
-        Grazing system and hilly topography will cost additional energy.
+        Grazing system and hilly topography will cos
```
Comment (JoeWaddell) [inline RUFAS/routines/animal/ration/animal_requirements.py]: Looks like an accidental keystroke here!

**Review by JoeWaddell — APPROVED** [review]
Once you fix the extra line in the docstring for `energy_activity_rqmts`, this looks great!

File: RUFAS/routines/animal/ration/animal_requirements.py (hunk 1 above)
Comment (Fjordrunner) [reply RUFAS/routines/animal/ration/animal_requirements.py]: Thanks for catching this, pushing now!

File: RUFAS/routines/animal/ration/animal_requirements.py [hunk 2]
```diff
@@ -441,6 +443,15 @@ def calc_rqmts(
             Average daily temperature of last month, °C
         average_daily_gain_heifer: float, optional
             Average daily gain of a heifer
+        NDF_conc:
+            Concentration (percent value) of a heifer's Neutral Detergent Fiber
```
Comment (JoeWaddell) [inline RUFAS/routines/animal/ration/animal_requirements.py]:
```suggestion
Concentration (percent value) of Neutral Detergent Fiber in previously fed ration.
```

File: RUFAS/routines/animal/ration/animal_requirements.py [hunk 3]
```diff
@@ -441,6 +443,15 @@ def calc_rqmts(
...
+        TDN_conc:
+            Concentration (percent value) of a heifer's Total Digestible Nutrients
+        net_energy_diet_concentration : float
```
Comment (JoeWaddell) [inline RUFAS/routines/animal/ration/animal_requirements.py]:
```suggestion
            Concentration (percent value) of Total Digestible Nutrients in previously fed ration.
```

**Review by JoeWaddell — COMMENTED** [review]
Just a couple small changes to the TDN and NDF descriptions (that should be copy/pasted to all of the TDN/NDF definitions).

File: RUFAS/routines/animal/ration/animal_requirements.py (hunk 3 above)
Comment (Fjordrunner) [reply RUFAS/routines/animal/ration/animal_requirements.py]: Just did this for both concentration parameters throughout the file, thanks for the help!

**Pull request merged**